airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -663,6 +765,16 @@ def execute(self, context) -> dict:
         self.log.info('Creating cluster: %s', self.cluster_name)
         hook = DataprocHook(gcp_conn_id=self.gcp_conn_id, impersonation_chain=self.impersonation_chain)
         # Save data required to display extra link no matter what the cluster status will be
+
+        if isinstance(self.cluster_config, str):
+            if urlparse(self.cluster_config).scheme == "gs":
+                self.cluster_config = self._build_cluster_config(self.cluster_config)
+            else:
+                self.log.info(
```

Comment:
Is it worth raising an exception here? If the user provides a non-GCS string then the call to `create_cluster` then I think that the call to `create_cluster` is going to fail, right?

In this case it might be preferable to raise here so that the DAG fails parsing rather than fails during execution.